continuous_integration/travis/install.sh

```diff
@@ -78,10 +78,10 @@ pip install --upgrade --no-deps \
 pip install --upgrade \
     cityhash \
     flake8 \
-    moto \
     mmh3 \
     pytest-xdist \
     xxhash
+pip install --upgrade git+https://github.com/spulec/moto  # includes fix for requests
```

Comment:
Is this not necessary on appveyor?  Do we test ORC on windows?

Reply:
We would test ORC wherever pyarrow.orc can be found; I don't know if they configure differently for windows.